mftf/2.3/metadata.md

```diff
@@ -410,7 +410,7 @@ Attribute|Type|Use|Description
 `name`|string|optional|Name of the operation.
 `dataType`|string|required|Data type of the operation. It refers to a `type` attribute of data entity that will be used as a source of input data.
 `type`|Possible values: `create`, `delete`, `update`, `get`.|required|Type of operation.
-`url`|string|optional |A routing URL of the operation. Example: `/V1/categories`. The full URL at the end will contain: `ENV.baseUrl` + /rest/ + `url`.
+`url`|string|optional |A routing URL of the operation. Example value: `"/V1/categories"`. The full URL at the end will contain: `ENV.baseUrl` + /rest/ + `url`.  To reference to an operation field of a particular operation type, wrap the reference into `{}` such as `{operationType.operation_field}`, or mere `{operation_field}` that will resolve the `operation_field` by the current or inherited `operationType`. Examples: `"/V1/bundle-products/{product2.sku}/links/{return}"`, or just `"/V1/bundle-products/{sku}/links/{return}"` as bundle-products's `operationType` is `product2`.
```

Comment:
@jilu1 I didn't find an attribute `operationType`. Did you mean `dataType` of the operation?